BD-4111: Add JSON parsing edge case for Context step (#9164)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/canvas/canvas_components/context.md

- The user will either advance to the next step or exit the Canvas if it’s the last step in the Canvas.
- In your Canvas step analytics, this will be counted as *Not Updated*.

### JSON types and Connected Content responses

> Braze evaluates context variables that are expected to be JSON (or Object)-type from Connected Content responses into strings. As a workaround, you'll need to enter these results into the this Liquid filter: `as_json_string`. An example is:

Reviewer: ```suggestion
Braze evaluates context variables that are expected to be JSON (or Object)-type from Connected Content responses into strings. To prevent context variables from being evaluated as strings, enter these results into this Liquid filter: `as_json_string`. An example is:
```
Would this edit still be accurate? 🤔 I'm not sure if I'm understanding correctly what's happening. We also might want to avoid using "workaround" language.
